README.md

```diff
@@ -6,6 +6,18 @@
 
 A simple wrapper around the [js-client-sdk](https://github.com/launchdarkly/js-client-sdk) that provides observable feature flags, a ready state to ensure all feature flags are up to date, and routing utilities.
 
+## Flag Types
+LaunchDarkly supports different flag types.
+
+- Boolean flags have two settings: `true` or `false`.
+- Multivariate flags have multiple settings that let you define different variations. Those settings include `string`, `number`, `JSON` `object`, or `JSON array`.
+
+## Use `invertFlag` with flags
+- **Boolean flags**:
+`false` by default. If set to `true`, the inverse of the requiredFlag's value will be used.
```

Comment:
nit: `false` is the default regardless of type.

```suggestion
`false` by default

- **Boolean flags**:
If set to `true`, the inverse of the requiredFlag's value will be used.
```

Reply:
For multivariate flag, even if the FF is OFF, you can serve non-Bool value.
[Reference](https://docs.launchdarkly.com/home/managing-flags/flag-variations)

You can return strings, numbers, JSON objects, or JSON arrays when the flag is OFF.

Reply:
Sorry, I mean that the default for `invertFlag` should always be `false`.

Reply:
Oops. my bad, misunderstood your words. I will update the text.